ansible/devutils

```diff
@@ -186,7 +249,10 @@ def action_pdu(parameters, action):
         if parameters['json']:
             data.append(status)
         else:
-            data.append([status[x] for x in header])
+            try:
+                data.append([status[x] for x in header])
+            except TypeError:
+                pass
```

Comment:
In my experience, if we got here, it is usually indicating problem with pdu manager/controller. We should surface the problem for debug/fix.

Reply:
Indeed. The excception is usually raised when some error happens in target function. I will cancel the ```try-except``` here.